src/main/ui/src/components/SearchByState.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {useState} from 'react'
+
+export default function SearchByState() {
+    const [data, setData] = React.useState([]);
+    const [searchTerm, setSearchTerm] = useState('');
+    React.useEffect(() => {
+    let url = `https://developer.nps.gov/api/v1/parks?parkCode="all"&limit=469&api_key=JMZizGv6gAcjBzoD4TbqW9RQSe9K8fEt9Cdb2Zta`;
+    fetch(url)
+        .then((response) => response.json())
+        .then((json) => {
+            setData(json.data);
+        })
+    .catch((error) => console.log(error));
+}, [searchTerm]);
+
+    function handleSearch(e) {
+        e.preventDefault();
+        setSearchTerm(e.target.value);
+    }
+
+    if (data.length > 0) {
+        let url = data.filter((i) => {
```

Comment:
VSCode error `'url' is declared but its value is never read.`

Reply:
url removed